contents/tutorials/test-frontend-feature-flags.md

```diff
@@ -131,35 +179,38 @@ When we run the app again, the main link on the page changed to "Go to PostHog."
 
```

Comment:
Here's an image to update the line above

https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_03_27_at_09_52_49_2x_65390628c1.png